android/clientmanager/src/main/java/io/mosip/registration/clientmanager/service/LoginService.java

```diff
@@ -54,7 +74,8 @@ public void saveAuthToken(String authResponse) throws Exception {
         byte[] decodedBytes = CryptoUtil.base64decoder.decode(cryptoResponseDto.getValue());
         try {
             JSONObject jsonObject = new JSONObject(new String(decodedBytes));
-            this.sessionManager.saveAuthToken(jsonObject.getString("token"));
+            List<String> roles=this.sessionManager.saveAuthToken(jsonObject.getString("token"));
+            return roles;
```

Comment:
when the roles are already part of shreadPreferences why do we need the list of roles to be returned.

Reply:
To be used as it is for now

Reply:
This work should be taken in the next sprint.